utils/constants.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+const (
+	// CleanupPriorityAnnotationKey is a annotation for the resources deployed in cluster ns which are waiting to
+	// be cleaned up after cluster is deleted.
+	// The value is an integer value [0,100], The larger the value, the later the order of deletion.
+	// The deletion order is :
+	// 1. delete resources without this annotation firstly.
+	// 2. delete resources with invalid value of this annotation (!= [0,100]).
+	// 3. delete resources following the priority value. For example, there are 2 manifestWorks, one value is set 100
+	// and another is set 10, the manifestWorks with 10 will be deleted before the one with 100.
+	CleanupPriorityAnnotationKey string = "open-cluster-management.io/cleanup-priority"
```

Comment:
this should also be in the types, but not here.

Reply:
moved to types of clusters